README.md

```diff
@@ -1095,6 +1095,25 @@ source ~/.bashrc
 cd $hotellogs
 ```
 
+## Saving your environment variables
+
+When you export FOO = "BAR" your variable is only exported in this section, to persist in the future you can simply append in your `~/.bash_profile` file the command to export your variable
+```bash
+echo export FOO="BAR" >> ~/.profile
```

Comment:
You talk about bash_profile and use profile here, both work but be consistent please.
Quotes are also irrelevant in this example and will be stripped in the file.